en/project_proposal_template.md

```diff
@@ -0,0 +1,73 @@
+# SCRF [Project Name] Proposal
+
+The [Smart Contract Research Forum](https://www.smartcontractresearch.org/) (SCRF) created a [PROJECT NAME] Proposal documenting a new project idea to implement at SCRF.
+
+## [Project Name] Overview
+
+Write 1-3 sentences giving an overview of the project and how it may bring value to its target audience.
+
+## Project Tracking
+
+Create a ticket or board in [GitHub](https://github.com/smartcontractresearchforum) to track the proejct. Link that here.
+
+## Relevant Documents
+
+Link any related documents for this project here. This may include process docs, templates, resources, pilot brainstorms, etc.
+
+Examples:
+* [SCRF Project Proposal - DevCon VI Research Community Hub](https://docs.google.com/document/d/1BC4uG4Bnwq1Gz56fzedV8o3Wb58-KJRTN3cV1-99nSo/edit?usp=sharing)
+* [SCRF Reading Group Project Proposal](https://docs.google.com/document/d/1BjsOH_9e1_gzFLoWCJrwnfAOq_Y5zFlREdhZNu7zEuE/edit?usp=sharing)
+* [SCRF.io Portal Project Plan](https://docs.google.com/document/d/1dZl-3wXLCW5N0OORUtV5CegMx7-TBYAp0ExbuBDSL0U/edit?usp=sharing)
+
```

Comment:
Not sure it's a great idea to link to google docs that are anon access and allow editing. Can we get these examples converted to markdown and put into the repo?